src/log-manager/src/nginx/nginx.conf.default

```diff
@@ -70,8 +70,18 @@ server {
     limit_except GET {
       deny all;
     }
-    default_type text/plain;
+    default_type application/octet-stream;
+    limit_rate_after 1m;
+    limit_rate 1m;
     access_by_lua_file /etc/nginx/lua/guard.lua;
     content_by_lua_file /etc/nginx/lua/get_log_content.lua;
   }
+
+  location @download_file {
+    internal;
+    add_header Accept-Ranges bytes;
+
+    root /;
```

Comment:
could this service serve `/usr/local/pai/logs/` directory only, and remove this prefix when processing?

Reply:
Fixed